src/main/java/homework/task1/Task1.java

```diff
@@ -0,0 +1,26 @@
+package homework.task1;
+
+public class Task1 {
+    public static void main(String[] args){
+
+        if(args.length > 0) {
+            for (int i = 0; i < args.length; i++) {
+
+                String toPrint = args[i];
+                if (toPrint.matches("error")){                                //checking for error
+                    System.out.println(ConsoleCollors.RED + "Alarm!" + ConsoleCollors.RESET);
+                    break;
+                }
+
+                System.out.println("Argument: " + toPrint +", "+ toPrint.length() + " letter(s)");
+            }
+        }
+    }
+    public static class ConsoleCollors{
```

Comment:
You don't need internal static class for holding constants. Just have them in Task1 class

Reply:
now i have an independent class in hw directory, done